cmd/limactl/disk.go

```diff
@@ -0,0 +1,307 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"text/tabwriter"
+
+	"github.com/docker/go-units"
+	"github.com/lima-vm/lima/pkg/qemu"
+	"github.com/lima-vm/lima/pkg/store"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newDiskCommand() *cobra.Command {
+	var diskCommand = &cobra.Command{
+		Use:   "disk",
+		Short: "Lima disk management",
+		Example: `  Create a disk:
+  $ limactl disk create DISK
+
+  List existing disks:
+  $ limactl disk ls
+
+  Delete a disk:
+  $ limactl disk delete DISK`,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+	}
+	diskCommand.AddCommand(
+		newDiskCreateCommand(),
+		newDiskListCommand(),
+		newDiskDeleteCommand(),
+		newDiskUnlockCommand(),
+	)
+	return diskCommand
+}
+
+func newDiskCreateCommand() *cobra.Command {
+	var diskCreateCommand = &cobra.Command{
+		Use: "create DISK",
+		Example: `
+To create a new disk:
+$ limactl disk create DISK
+`,
+		Short: "Create a Lima disk",
+		Args:  cobra.ExactArgs(1),
+		RunE:  diskCreateAction,
+	}
+	diskCreateCommand.Flags().String("size", "2G", "configure the disk size")
```

Comment:
Isn't this too small?

Reply:
> Isn't this too small?

I agree that it sounds small, but I don't know how to pick a reasonable default size.

Given that this is just the maximum size of a sparse file, there is not much harm in picking a larger value, but which size should be "large enough" for most use cases?

Maybe `10G`? But it still is pretty arbitrary.

Reply:
Maybe half the default size of the primary disk? `50G`?

Reply:
Every disk needs a size, why not make it empty and make it a required flag?